src/injected/client-utils.ts

```diff
@@ -41,10 +41,14 @@ export class ClientUtils {
     }
 
     public matchesSelector(element: ElementMatcher, selectorString: string): boolean {
```

Comment:
I think we should just get rid of this method entirely; it's an IE9 polyfill for a method that Chromium has supported since 2014, and we already have other parts of the code using it without polyfilling. I think it would be fine to just update the only callsite (`heading-formatter.ts`) to invoke `element.matches(selector)` directly in place of `clientUtils.matchesSelector(element, selector)`.

(Alternately, we could add an `HtmlElementUtils` dependency in `heading-formatter` and use the existing non-polyfill wrapper there (`elementMatches`); I'm not sure that would add any value to `heading-formatter`, since it already depends directly on a variety of other `Element` methods/properties directly)